### PR TITLE
JDK11 required for Android mobile v2 development

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-43226

### CODE REVIEW

File: site/content/contribute/more-info/mobile/developer-setup/_index.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ If you run into any issues getting your environment set up, check the {{< newtab
 A macOS computer is required to build the Mattermost iOS mobile app.
 {{</note>}}
 
+{{<note "Android mobile app">}}
+JDK11 is required to develop for v2 of the Android mobile app. You can download the latest OpenJDK release of Java from Oracle, for free, under an open source license.
```
Comment (cwarnermm) [inline site/content/contribute/more-info/mobile/developer-setup/_index.md:22]:
```suggestion
JDK11 is required to develop for v2 of the Mattermost Android mobile app. You can download the latest OpenJDK release of Java from Oracle, for free, under an open source license.
```

**Comment by azigler** [comment]
Hi @cwarnermm! I have two questions about this one:

1) Since this Jira ticket is from last year and mobile v2 has since been released and is the primary version, do we need to specify that this is to "develop for v2"? Can we just say it's "to develop", in this case now?
2) Can they use a JDK later than 11, or does it specifically have to be 11? The ticket and Community thread are inconclusive, but we should make sure the documentation is clear. JDK11 is old, I would assume most developers have some form of the JDK but likely later versions. If that's a problem, we should indicate that clearly.

**Comment by cwarnermm** [comment]
@enahum - Can you help answer Andrew's questions, please?
- Do we want to continue to make a clear distinction between mobile v1 and v2 for developers?
- Does v2 specifically need JDK 11, or is a range of releases supported?

**Comment by enahum** [comment]
> @enahum - Can you help answer Andrew's questions, please?
> 
> - Do we want to continue to make a clear distinction between mobile v1 and v2 for developers?

No need for v1 anymore


> - Does v2 specifically need JDK 11, or is a range of releases supported?

Yes, JDK11

**Comment by cwarnermm** [comment]
Thanks, @enahum! Can users build a custom mobile app using the v2 codebase?

**Comment by enahum** [comment]
Well that is the idea behind this documentation :)

File: site/content/contribute/more-info/mobile/developer-setup/_index.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ If you run into any issues getting your environment set up, check the {{< newtab
 A macOS computer is required to build the Mattermost iOS mobile app.
 {{</note>}}
 
+{{<note "Android mobile app">}}
+JDK11 is required to develop for v2 of the Mattermost Android mobile app. You can download the latest OpenJDK release of Java from Oracle, for free, under an open source license.
```
Comment (cwarnermm) [inline site/content/contribute/more-info/mobile/developer-setup/_index.md:22]:
```suggestion
JDK11 is required to develop the Mattermost Android mobile app. You can download the latest OpenJDK release of Java from Oracle, for free, under an open source license.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**